.phpcs.xml.dist

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress IMporter">
+	<description>The code standard for the WordPress Importer plugin.</description>
+
+	<!-- What to scan -->
+	<file>.</file>
+	<exclude-pattern>vendor/</exclude-pattern>
+
+	<!-- How to scan -->
+	<arg value="sp"/> <!-- Show sniff and progress -->
+	<arg name="basepath" value="."/><!-- Strip the file paths down to the relevant bit -->
+	<arg name="colors"/>
+	<arg name="extensions" value="php"/>
+	<arg name="report" value="full"/>
+	<arg name="report" value="summary"/>
+	<arg name="report" value="source"/>
```

Comment:
Generally speaking, `report` is one of those arguments you would normally not put in the ruleset to allow devs more flexibility.

If you do wish to keep it there, you can merge the three commands (PHPCS 3.x):
```xml
<arg name="report" value="full,summary,source"/>
```